README.md

```diff
@@ -742,6 +742,34 @@ You will find information on the repair operation in the `LOG` file inside the d
 
 Before calling `repair()`, close a database if it's using the same `location`.
 
+## Ordering
+
+All write operations are asynchronously dispatched on the node thread pool and therefore the ordering
+is not guaranteed.
+
+Consider:
+
+``js
+await Promise.all([
+  db.put('foo', 1)
+  db.put('foo', 2)
+])
+const result = await db.get('foo)
+```
+
+The value of `result` could be either `1` or `2` depending on thread pool scheduling.
+
+Furthermore since reads fetch the snapshot immediatly, writes will not be visible until at latest
```

Comment:
This is not entirely true: https://github.com/Level/leveldown/issues/796 (same applies to `classic-level`).

Let's leave this paragraph (and the example) out for now, because I fear it will only create more confusion, at least until we fix https://github.com/Level/leveldown/issues/796 (or decide that the current behavior is desirable, maybe).

Reply:
Agreed with @vweevers :)